code/game/world.dm

```diff
@@ -435,11 +435,12 @@ var/world_topic_last = world.timeofday
 
 #undef SET_THROTTLE
 
-/world/Reboot(var/reason)
+/world/Reboot(reason, hard_reboot = FALSE)
 	if(GLOB.using_map.reboot_sound)
 		sound_to(world, sound(pick(GLOB.using_map.reboot_sound)))// random end sounds!! - LastyBatsy
 
-	Master.Shutdown()
+	if(!hard_reboot)
```

Comment:
I don't think this should be exposed to admins.

There shouldn't be any reason for them to need it, and the code assumes that Shutdown() will run if the round ends without daemon crashing.

Reply:
I agree, if the server is entering a state where it can't be rebooted, that's an issue that should be solved, not something to paper around with something like this.